### ISSUE TITLE
Data Collector Processes and Threads per Process settings not taking effect

### ISSUE THREAD

**Issue by dsclassen** [Cacti/spine#251]
**Describe the bug**

Changes to the poller Processes or Threads are not getting implemented. During this ~20 minute window I changed the values of Data Collector Processes and Threads per Process from 1/1 to 12/12 and everything in-between, yet the log files indicate that the Processes and Threads are always 1.

```
2022-03-16 16:11:59 - SYSTEM STATS: Time:59.4620 Method:cmd.php Processes:1 Threads:1 Hosts:33 HostsPerProcess:33 DataSources:431 RRDsProcessed:360
--
2022-03-16 16:10:59 - SYSTEM STATS: Time:59.5377 Method:cmd.php Processes:1 Threads:1 Hosts:33 HostsPerProcess:33 DataSources:431 RRDsProcessed:360
2022-03-16 16:09:59 - SYSTEM STATS: Time:59.4682 Method:cmd.php Processes:1 Threads:1 Hosts:33 HostsPerProcess:33 DataSources:431 RRDsProcessed:2
2022-03-16 16:08:05 - SYSTEM STATS: Time:5.6238 Method:spine Processes:1 Threads:1 Hosts:33 HostsPerProcess:33 DataSources:431 RRDsProcessed:3
2022-03-16 16:07:06 - SYSTEM STATS: Time:5.3637 Method:spine Processes:1 Threads:1 Hosts:33 HostsPerProcess:33 DataSources:431 RRDsProcessed:3
2022-03-16 16:06:06 - SYSTEM STATS: Time:5.3509 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:05:06 - SYSTEM STATS: Time:5.3496 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:04:06 - SYSTEM STATS: Time:5.3517 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:03:06 - SYSTEM STATS: Time:5.3410 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:02:06 - SYSTEM STATS: Time:5.3343 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:01:07 - SYSTEM STATS: Time:6.3459 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 16:00:06 - SYSTEM STATS: Time:5.3400 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 15:59:06 - SYSTEM STATS: Time:5.3258 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 15:58:06 - SYSTEM STATS: Time:5.3174 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:3
2022-03-16 15:57:06 - SYSTEM STATS: Time:5.5897 Method:spine Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:361
2022-03-16 15:57:00 - SYSTEM STATS: Time:59.8340 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
2022-03-16 15:56:00 - SYSTEM STATS: Time:59.8396 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
2022-03-16 15:55:00 - SYSTEM STATS: Time:59.7997 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
2022-03-16 15:54:00 - SYSTEM STATS: Time:59.8237 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
2022-03-16 15:53:00 - SYSTEM STATS: Time:59.8360 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
2022-03-16 15:52:00 - SYSTEM STATS: Time:59.8334 Method:cmd.php Processes:1 Threads:1 Hosts:34 HostsPerProcess:34 DataSources:433 RRDsProcessed:360
```
**Expected behavior**
I would expect the Time and the reported numbers of Processes and Threads to change as I fiddle with the Process and Thread settings..

**Server (please complete the following information):**
 - OS: Rocky Linux 8.5
 - Cacti: 1.2.20

**Comment by dsclassen** [comment]
I see that this is now edited in the settings for the individual Data Collectors. I'll close this now. Sorry for the false alarm.

**Comment by TheWitness** [comment]
Yea, I need to hide those default Scott.  They don't add any value and people who come from 1.1.x or 0.8.x get confused quickly.

**Comment by dkuji** [comment]
@dsclassen @TheWitness 
The same event is occurring in my environment.

What does this mean?
Does it mean that even if I change the number of processes and threads, the log always outputs 1 process and 1 thread?

**Comment by dkuji** [comment]
@TheWitness 
Thanks.

I understood that the preset(like a default value ?) is only displayed and that it is actually running in the process and thread specified in the data collector.

**Comment by TheWitness** [comment]
Right.

**Comment by dkuji** [comment]
@TheWitness 

I am using Spine for my data collector.
I have changed the number of script servers, processes and threads from 1/1/1 to 2/2/10.
But the SYSTEM STATS times in cacti.log remain the same.

The data collector settings have been implemented as shown in the attached image, is this correct?

OS: Rocky Linux 8.6
Cacti: 1.2.21-1

<img width="698" alt="ScreenShot 2022-11-09 10 31 10" src="https://user-images.githubusercontent.com/22391144/200716201-b94c82f1-b9ac-4bc7-919e-7a391457fd91.png">
